civihr_default_theme/templates/node/webform-calendar.tpl.php

```diff
@@ -3,9 +3,19 @@
 ?>
 <input
   type="text"
-  data-date-format="dd/mm/yyyy"
+  required="<?php $component['required']; ?>"
   id="edit-submitted-<?php print $idKey ?>"
   class="form-text <?php print implode(' ', $calendar_classes); ?>"
   alt="<?php print t('Open popup calendar'); ?>"
   title="<?php print t('Open popup calendar'); ?>" />
+<input
+  type="date"
+  required="<?php $component['required']; ?>"
+  id="edit-submitted-<?php print $idKey ?>"
```

Comment:
wouldn't this create two input elements with the exact same ID? IDs should be unique.

Reply:
`edit-submitted-<?php print $idKey ?>`
`edit-submitted-<?php print $idKey ?>-mobile` ?

Reply:
Yes they would, and the ID needs to be same, but I am removing one of the element based on device.
https://github.com/compucorp/civihr-employee-portal-theme/blob/556322b2ee516a4d8671320d4a43a082aca85c7f/civihr_default_theme/assets/js/civihr_default_theme.script.js#L182